docs/tutorials/active_learning_with_small_text.ipynb

```diff
@@ -77,7 +77,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%pip install \"rubrix[listeners]\" datasets \"small-text\" \"transformers[torch]\""
+    "%pip install \"rubrix[listeners]\" datasets \"small-text>=1.1.0\" \"transformers[torch]\""
```

Comment:
Using older `datasers` versions, the downloaded dataset has different feature names. I think that is why the settled revision finds different dataset versions for both of us.

```suggestion
    "%pip install \"rubrix[listeners]\" \"datasets~=2.5.0\" \"small-text>=1.1.0\" \"transformers[torch]\""
```

Reply:
I noticed the problem with the feature names, but using the `revision` argument still loaded the previous dataset version for me. And this resulted in a situation where I had the old dataset with your code for the new attribute name which again lead to that attribute error (in exactly the opposite way as before). This occurred even after clearing the cache, the data was definitely re-downloaded. I can't rule out an error on my side here but this is the reason why I switched to `version`.